views/notfound.php

```diff
@@ -29,18 +29,24 @@
 			}
 
 			.w720 {
-				width: 720px;
+				width: 90%;
+				max-width: 720px;
 				margin: 0px auto;
 			}
 
+			@media (max-width: 480px) {
+				iframe {
+					height: 1300px !important;
```

Comment:
Seems this would greatly exceed the device height?

Reply:
On mobile, the landscape design switches to a portraitversion. But the iframe isn't high enough. That's why the rule is there.